ratom/score.go

```diff
@@ -121,6 +127,28 @@ func Clone(repo string) string {
 	}
 	return dir
 }
+
+func runGraphAPI(url string) string{
+	fmt.Println(url)
+	command := "python3 ratom/metrics/GraphQL.py " + url
+	r := subprocess.New(command, subprocess.Shell)
+	r.Exec()
+
+	dat, err := os.ReadFile("QueryOutput.txt")
+	if err != nil {
+		fmt.Println("PANICCCCCC")
```

Comment:
I like the comment, but maybe add what is failing lol, ex:
```suggestion
		fmt.Println("PANICCCCCC:", err)
```